sirepo_bluesky/sirepo_ophyd.py

```diff
@@ -92,8 +95,10 @@ def __init__(
         sim_type = self.connection.data["simulationType"]
         allowed_sim_types = ("srw", "shadow")
         if sim_type not in allowed_sim_types:
```

Comment:
should 'madx' be an allowed simulation type?

Reply:
Good point, thanks for catching this, @jklynch!